azure-devops/projects/selfcare-projects/tlscert-dev-dev-selfcare-pagopa-it.tf

```
@@ -0,0 +1,75 @@
+variable "tlscert-dev-dev-selfcare-pagopa-it" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert         = true
+      path                    = "TLS-Certificates\\DEV"
+      dns_record_name         = ""
+      dns_zone_name           = "dev.selfcare.pagopa.it"
+      dns_zone_resource_group = "selc-d-vnet-rg"
+      # common variables to all pipelines
+      variables = {
+        CERT_NAME_EXPIRE_SECONDS = "2592000" #30 days
+        KEY_VAULT_NAME           = "selc-d-kv"
+      }
+      # common secret variables to all pipelines
+      variables_secret = {
+      }
+    }
+  }
+}
+
+locals {
+  tlscert-dev-dev-selfcare-pagopa-it = {
+    tenant_id         = module.secrets.values["PAGOPAIT-TENANTID"].value
+    subscription_name = "DEV-SelfCare"
+    subscription_id   = module.secrets.values["PAGOPAIT-DEV-SELFCARE-SUBSCRIPTION-ID"].value
+  }
+  tlscert-dev-dev-selfcare-pagopa-it-variables = {
+    KEY_VAULT_CERT_NAME          = "${replace(var.tlscert-dev-dev-selfcare-pagopa-it.pipeline.dns_record_name, ".", "-")}-${replace(var.tlscert-dev-dev-selfcare-pagopa-it.pipeline.dns_zone_name, ".", "-")}"
+    KEY_VAULT_SERVICE_CONNECTION = module.DEV-SELFCARE-TLS-CERT-SERVICE-CONN.service_endpoint_name
+  }
+  tlscert-dev-dev-selfcare-pagopa-it-variables_secret = {
+  }
+}
+
+module "tlscert-dev-dev-selfcare-pagopa-it-cert_az" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_tls_cert?ref=v1.1.0"
```

```suggestion
  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_tls_cert?ref=v2.0.0"
```
for apex domain we need to use 2.0.0 version